components/function-runtimes/python39/kubeless/README.md

```diff
@@ -0,0 +1,24 @@
+## How to locally run and debug Python Function nad runtime
+
+* Create [venv](https://docs.python.org/3/library/venv.html)
```

Comment:
These steps must be done in this order, right?
If so, can you change the unordered list to the ordered one?